site/docs/hosting/cloudflare-workers.md

```diff
@@ -0,0 +1,121 @@
+# Hosting: Cloudflare Workers
+
+[Cloudflare Workers](https://workers.cloudflare.com/) is a public serverless computing platform that offers a convenient and simple solution for running JavaScript at the edge.
+With the ability to handle HTTP traffic and work with [Service Worker API](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API), building Telegram bots becomes a breeze.
+In addition, you can even develop [Web Apps](https://core.telegram.org/bots/webapps) at the edge, all for free within certain quotas.
+
+This guide will take you through the process of hosting your Telegram bots on Cloudflare Workers.
+
+## Prerequisites
+
+To follow along, please make sure that you have a [Cloudflare account](https://dash.cloudflare.com/login) with your workers subdomain [configured](https://dash.cloudflare.com/?account=workers).
+
+## Setting Things Up
+
+First, create a new project:
+
+```sh
+npx wrangler generate my-bot
```

Comment:
Should we tell people to install the CLI first?

Reply:
The CLI?

Reply:
Yes. `wrangler`. We didn't mention how to set up the system.

Reply:
Setup the system as in installing `wrangler`?

Reply:
Yes

Reply:
It does not require any installation. `npx` runs it directly.

Reply:
Ohhhh right, `npx` magic. Let's leave it then.